### DISCUSSION TITLE
Box elements in React

### DISCUSSION THREAD

**Discussion by cbix** [primer/react#1343] [Q&A]
Hello :) I just started moving a React project to using Primer (unfortunately, the first challenge was that we're stuck at 22.0.3 for now until Enzyme supports React 17, which is required for later versions of Primer ...).

Now I'd like to use Box elements described [here](https://primer.style/css/components/box) like `Box-header` and `Box-footer` but realized these do not exist in `@primer/components`.

What's the best way to use these Box elements in React or is this a WIP?

**Comment by colebemis** [comment]
Hi, @cbix. We don't have the equivalent of the Primer CSS `Box` component in Primer React yet.
